editor/app/src/main/java/ca/ubc/eml/soiltopargraphy/editor/ui/quizpanel/QuestionnairePanelFragment.java

```diff
@@ -71,4 +74,13 @@ public void nextQuestion(View view) {
         QuestionnairePanel newSoilType = new QuestionnairePanel(soilType, color, density, fruitfulness, humidity);
     }
 
-}
+
+    public void onClick(View v) {
+        Fragment nextFragment = new FlagMapFragment();
+        FragmentManager fragmentManager = getActivity().getSupportFragmentManager();
+        android.support.v4.app.FragmentTransaction transaction = fragmentManager.beginTransaction();
```

Comment:
You should import `android.support.v4.app.FragmentTransaction` so that you can just write 
```java
FragmentTransaction transaction = fragmentManager.beginTransaction();
```

Alternatively, you can replace these 3 lines with a chain:
```java
getActivity().getSupportFragmentManager()
    .beginTransaction()
    .replace(R.id.container, nextFragment)
    .commit();
```